docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Environment Variables"
+---
+
+Environment variables are useful when:
```

Comment:
Or a link to https://en.wikipedia.org/wiki/Environment_variable

Also:
Most of the features in Oryx are driven by data coming from a backend. There are however some exceptions in case we need to deploy an environment with hardcode variables that cannot be retrieved from a backend for different reasons:

Reply:
Added a link to the wiki page, can't get the idea behind that 'Also: block' - should we change/add something related to that?